### PR TITLE
Add new test case for kernel config verify

### PR DESCRIPTION
**Pull request by prashantmhetar**
This test compares the kernel configuration file with master.config stored in blob. 
The latest OL image have config file embedded so modified this PR to support it.

### CODE REVIEW

**Comment by prashantmhetar** [comment]
[LISAv2 Test Results Summary]
Test Run On           : 01/31/2019 12:20:51
ARM Image Under Test  : Canonical : UbuntuServer : 18.04-LTS : latest
Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
Total Time (dd:hh:mm) : 0:0:26

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 KERNEL-CONFIG-VERIFY-TEST                                                         PASS                 6.02

File: Testscripts/Linux/kernel_config_verify_test.sh [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+#To download the config files
+download_config_files()
+{
+	#Get the master config file
+	wget $MasterConfigUrl -O $MasterConfigFile
+	
+	#Get the image config file
+	wget $ImageConfigUrl -O $ImageConfigFile
+
```
Comment (chvalean) [inline Testscripts/Linux/kernel_config_verify_test.sh]: remove blank spaces

File: Testscripts/Linux/kernel_config_verify_test.sh [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+}
+
+#######################################################################
+#
+# Main script body
+#
+#######################################################################
+
+#Initially get the kernel config files
```
Comment (chvalean) [inline Testscripts/Linux/kernel_config_verify_test.sh]: add a blank space after # and the actual comment. applicable to all comments

File: XML/TestCases/FunctionalTests-OL.xml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    <test>
+        <TestName>KERNEL-CONFIG-VERIFY-TEST</TestName>
+        <testScript>KERNEL-CONFIG-VERIFY-TEST.ps1</testScript>
+        <files>.\Testscripts\Linux\utils.sh,.\Testscripts\Linux\kernel_config_verify_test.sh</files>
+        <setupType>OneVM</setupType>
+        <TestParameters>
+            <param>MasterConfigUrl=MASTER_KERNEL_CONFIG_URL</param>
+	    <param>ImageConfigUrl=IMAGE_KERNEL_CONFIG_URL</param>
+        </TestParameters>
+        <Platform>Azure,OL</Platform>
```
Comment (chvalean) [inline XML/TestCases/FunctionalTests-OL.xml]: Only OL maybe?

**Comment by chvalean** [comment]
@prashantmhetar please also check and fix the PR Checker failures

File: Testscripts/Linux/kernel_config_verify_test.sh (hunk 1 above)
Comment (prashantmhetar) [reply Testscripts/Linux/kernel_config_verify_test.sh]: removed the blank spaces in this file

File: Testscripts/Linux/kernel_config_verify_test.sh (hunk 2 above)
Comment (prashantmhetar) [reply Testscripts/Linux/kernel_config_verify_test.sh]: Added a blank space after # in comments

File: XML/TestCases/FunctionalTests-OL.xml (hunk 3 above)
Comment (prashantmhetar) [reply XML/TestCases/FunctionalTests-OL.xml]: set Platform as OL only

File: Testscripts/Windows/KERNEL-CONFIG-VERIFY-TEST.ps1 [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+        try {
+            Get-RemoteFileInfo -filename $imageconfigPath  -server $HvServer
+        }
+        catch {
+            Write-LogErr "The .config file $imageconfigPath could not be found!"
+            throw
+        }
+
+        Write-LogInfo "Copy the config files to Remote VM"
+	# remote copy the config files to remote Virtual machine
```
Comment (LiliDeng) [inline Testscripts/Windows/KERNEL-CONFIG-VERIFY-TEST.ps1]: why not wget from VM directly?

Comment (prashantmhetar) [reply Testscripts/Windows/KERNEL-CONFIG-VERIFY-TEST.ps1]: yes, modified the script to use wget

Comment (prashantmhetar) [reply Testscripts/Windows/KERNEL-CONFIG-VERIFY-TEST.ps1]: Re-checked for wget using OL qemu image. It is unable to download the file from blob.

File: Testscripts/Windows/KERNEL-CONFIG-VERIFY-TEST.ps1 [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+# Licensed under the Apache License.
+<#
+.Synopsis
+    Verify the kernel config file.
+
+.Description
+    This script will checks the kernel config file for given image and
+    compare it with master config file.
+
+.Parameter vmName
```
Comment (chvalean) [inline Testscripts/Windows/KERNEL-CONFIG-VERIFY-TEST.ps1]: remove details for params vmname, hvserver and testparams, unused in this format.

**Pull request merged**